lib/how_is/chart.rb

```diff
@@ -4,6 +4,15 @@ def self.gnuplot(commands)
   end
 
   def self.rotate(offset, filename)
+    if on_windows?
```

Comment:
Probably better to use `Gem.win_platform?` than defining a method for it. I'll look at the PR in more detail later. (Probably in a few days.)